packages/experiments/src/components/Button/Button.types.tsx

```diff
@@ -66,13 +66,19 @@ export interface IButton {
   focus: () => void;
 }
 
+export type INativeButtonProps = Pick<
+  React.AllHTMLAttributes<HTMLAnchorElement | HTMLButtonElement | HTMLDivElement>,
+  Exclude<keyof React.AllHTMLAttributes<HTMLAnchorElement | HTMLButtonElement | HTMLDivElement>, 'content'>
+>;
```

Comment:
You have just written `Omit`. 😄 It will be a built-in type as of TS3.5 but there is one you can import from utilities for now. You should be able to do:
```tsx
export type INativeButtonProps = Omit<React.AllHTMLAttributes<HTMLAnchorElement | HTMLButtonElement | HTMLDivElement>, 'content'>;
```